Library/Formula/natalie.rb

```diff
@@ -8,14 +8,15 @@ class Natalie < Formula
   depends_on :xcode => "6.3"
 
   def install
-    bin.install "natalie.swift"
+    system "xcrun -sdk macosx swiftc -O natalie.swift -o natalie"
+    bin.install "natalie"
```

Comment:
(you can pass multiple arguments to install, like `bin.install "natalie", "natalie.swift"`)